scripts/make_generated_files.py

```diff
@@ -205,6 +213,15 @@ def check_generated_files(generation_scripts: List[GenerationScript], root: Path
     for generation_script in generation_scripts:
         for file in generation_script.files:
             file = root / file
+            if not file.exists():
```

Comment:
Any reason we don't use the same logic as bellow?

`if generation_script.optional and not bak_file.exists():`